src/features/services/queries.ts

```diff
@@ -136,3 +134,32 @@ export function useQueryValues(keys: string[]) {
     })),
   });
 }
+
+export function useTransferKey() {
+  // eslint-disable-next-line
+  const [_, network] = useNetworkContext();
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationFn: async (key: string) =>
+      await network?.kvStore.transfer({ key, newOwner: "maiyg" }),
```

Comment:
Assigning the owner to `maiyg` makes it immutable, as that identity is reserved by Many to be no one.